ENIGMAsystem/SHELL/Graphics_Systems/OpenGL1/GLvertex.cpp

```diff
@@ -113,19 +113,11 @@ bool vbo_is_supported = false;
 void graphics_init_vbo_method() {
   // we don't check for extensions until GLEW has been initialized
   vbo_is_supported = GLEW_ARB_vertex_buffer_object;
-  // if the vbo extension is supported, but the GL version is old,
-  // we need to alias the buffer functions to the extension ones
-  if (GLEW_ARB_vertex_buffer_object == true && GLEW_VERSION_1_5 == false) {
```

Comment:
This change boils down to you making this `if` always be true, yes? So why not just do that? Or, specifically, why not just do the thing I suggested at the time and just set these methods if the ARB version isn't null?

Reply:
Well.... why keep the condition and the extra 8 lines? The idea here is to use the ARB version always. However, Rusky disputes me that it is totally identical to the non-ARB versions of the functions. I think he may be right about that and I was investigating for some facts in the spec, but haven't found anything yet.

Also, I suppose we could change the existing version to look at each function individually, but I see no reason to do that Josh. With OpenGL extensions, this is an all or none kind of thing. So we really don't need a condition on each vertex buffer function. The way I have it makes the most sense.

Reply:
I assumed you were using this change to demonstrate a clear difference between the two on some system (specifically, I was thinking some Travis weirdism was involved). If that's not the case, then no, I'd rather not do this...

Reply:
Yeah no, there doesn't seem to be any observed behavioral difference here at all. And if that's the case, please block the pull request by disapproving it. Thanks.